demoScripts/vmssQuickstart.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Quickstart - Create a Virtual Machine Scale Set with Azure CLI
+description: Get started with your deployments by learning how to quickly create a Virtual Machine Scale Set with Azure CLI.
+author: ju-shim
+ms.author: jushiman
+ms.topic: quickstart
+ms.service: virtual-machine-scale-sets
+ms.date: 11/22/2022
+ms.reviewer: mimckitt
+ms.custom: mimckitt, devx-track-azurecli, mode-api
+---
+
+# Quickstart: Create a Virtual Machine Scale Set with the Azure CLI
+
+**Applies to:** :heavy_check_mark: Linux VMs :heavy_check_mark: Windows VMs :heavy_check_mark: Uniform scale sets
+
+> [!NOTE]
+> The following article is for Uniform Virtual Machine Scale Sets. We recommend using Flexible Virtual Machine Scale Sets for new workloads. Learn more about this new orchestration mode in our [Flexible Virtual Machine Scale Sets overview](flexible-virtual-machine-scale-sets.md).
+
+A Virtual Machine Scale Set allows you to deploy and manage a set of auto-scaling virtual machines. You can scale the number of VMs in the scale set manually, or define rules to autoscale based on resource usage like CPU, memory demand, or network traffic. An Azure load balancer then distributes traffic to the VM instances in the scale set. In this quickstart, you create a Virtual Machine Scale Set and deploy a sample application with the Azure CLI.
+
+[!INCLUDE [quickstarts-free-trial-note](../../includes/quickstarts-free-trial-note.md)]
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment.md)]
+
+- This article requires version 2.0.29 or later of the Azure CLI. If using Azure Cloud Shell, the latest version is already installed. 
+
+
+## Define Environment Variables
```

Comment:
Do we consider these self-documenting? I think we can but we should at least call that out in the text, something like:

Throughout this document we use environment variables to facilitate cut and paste reuse from this document. The default values below will enable you to work through this document in most cases. The meaning of each environment variable will be addressed as they are used in the steps below.

Reply:
Agreed - This is the text I settled on with Cynthia, happy to take your input though

"Environment variables are commonly used in Linux to centralize configuration data to improve consistency and maintainability of the system. Create the following environment variables to specify the names of resources that will be created later in this tutorial:"

Reply:
Why are we teaching "Grandmother to suck eggs"?

If our target audience is Linux Savvy people then shouldn't we talk to them as if they are Linux Savvy? This feels like we are talking to a Microsoft person who doesn't know how to use Linux. I'm going to put my suggested text in and we can discuss further.

Reply:
You are the expert here so we should definitely iterate on what we say. I think the balance we need to strike is that unfortunately not everyone who lands on this page will be 'Linux Savvy'. I believe this will be particularly true as we see a shift from Windows to Linux.

Reply:
I think we should target the people we want to win, rather than the people who will gravitate to wherever it is easiest. If there is a need to educate Windows people that is a completely different initiative. We should target Azurisms and Windowisms, one of which is treating everyone as if they are a Windows person.